.idea/markdown-navigator-enh.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
To avoid we comminting this files could you please update our `.gitignore`?

Remove this lines:
```
.idea/caches
.idea/dictionaries
.idea/libraries
.idea/modules
.idea/compiler.xml
.idea/gradle.xml
.idea/jarRepositories.xml
.idea/misc.xml
.idea/vcs.xml
.idea/workspace.xml
.idea/modules.xml/
.idea/assetWizardSettings.xml
```
And add this ones:
```
.idea/*
!.idea/codeStyles
```